### PR TITLE
Upgrading guide - finishing touches

### PR DESCRIPTION
**Pull request by adamlazik1**
Removed reference to EL 7 and edited Release notes URL based on checking all the URLs in the upgrading guide for the upcoming release of Satellite 6.12.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7
* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc
Comment (adamlazik1) [inline guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc:11]: I do not know whether the procedure in this file is the same for EL 8. Is there someone who could confirm this?

Comment (Lennonka) [reply guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc:11]: "Cloning to the Target Server", step 4. should probably enable RHEL8 packages. Otherwise I wonder if all stays the same, I'm trying to check.

**Review by Lennonka — COMMENTED** [review]
Mostly looks good. Added some remarks.

File: guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc
Comment (adamlazik1) [reply guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc:11]: Thanks so much for checking! I'll ask in the satellite IRC channel, too

**Comment by adamlazik1** [comment]
I took repositories from the fresh satellite RHEL 8 installation: https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/installing_satellite_server_in_a_connected_network_environment/installing_server_connected_satellite#anchor_xml_id_repositories_rhel_8_xreflabel_repositories_rhel_8_red_hat_enterprise_linux_8

**Comment by adamlazik1** [comment]
Since the content of the first two commits has been approved, I am merging those via another PR. It is good to get as many fixes as possible before the Satellite 6.12 release.

File: guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc
Comment (maximiliankolb) [inline guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc:38]:
```suggestion
Do not install {EL} 8 software groups or third-party applications.
```

File: guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc
Comment (maximiliankolb) [inline guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc:39]: Alternative:

```suggestion
Ensure that your server complies with all the required specifications.
For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_.
```

I am unsure if my suggestions adds a benefit apart from consistency (which is a benefit in itself). Your call.

File: guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc
Comment (maximiliankolb) [inline guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc:11]:
```suggestion
Instead, you must backup the existing {SmartProxyServer}, restore it on {EL} 8, and then reconfigure {SmartProxyServer}.
```

**Comment by adamlazik1** [comment]
@maximiliankolb  Thank you for your feedback. Incorporated the suggested changes.

**Review by maximiliankolb — APPROVED** [review]
LGTM; handing over to Lena if she has any further comments. Else feel free to merge right away.

**Comment by adamlazik1** [comment]
Merged and cherry-picked:
   544edc029..b087eb8fe  3.3 -> 3.3
   52c328adc..11e75a21b  3.4 -> 3.4
   a88a23be9..509be8f45  3.5 -> 3.5

**Pull request merged**
